index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="de">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <nav>
+        <span>Glücklich am Deich</span>
+        <H1>Ferienhof Struve</H1>
+        <ul>
+            <li>Ferienwohnung</li>
+            <li>Ferienhaus</li>
+            <li>Unser Hof</li>
+            <li>Preise / Buchen</li>
+            <li>Umgebung</li>
+            <li>Kontakt</li>
+        </ul>
+    </nav>
+    <main>
+        <section>
+        <h2>Willkommen auf unserem Hof!</h2>
+        <a href="#house">
+            <img src="https://www.ferienwohnung-eiderstedt-nordsee.de/images/traumurlaub%20am%20deich.jpg?crc=4257483147" alt="Ferienhaus Opa Hans">
+        </a>
+        <a href="#apartment">
+            <img src="https://www.ferienwohnung-eiderstedt-nordsee.de/images/essbereich.jpg?crc=3882799318" alt="Ferienwohnung Kuschelnest">
+        </a>
+    </section>
+    <section>
+        <h3 id="house">Ferienhaus Opa Hans</h3>
+        <p>Ferien am Deich! Fernab von allem Trubel ohne Nachbarn. Großzügiges und barrierefreies Ferienhaus in einmaliger Alleinlage mit Topausstattung. Vom Deutschen Tourismus mit 5 Sternen ausgezeichnet. Strand nur 2500 m entfernt.</p>
+        <a href="">Mehr zu Opa Hans</a>
+    </section>
+    <section>
+        <h3 id="apartment">Ferienwohnung Kuschelnest</h3>
+        <p>Herzlich willkommen im Kuschelnest für Zwei. Die Wohnung befindet sich in unserem ehemaligen Pferdestall, den wir liebevoll für Sie umgebaut haben.</p>
+        <a href="">mehr zum Kuschelnest</a> 
+    </section>
+    </main>
+</body>
+</html>
```

Comment:
Newline is missing. Check out https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline